CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 
 ## [2.8.0] - 2017-10-18
 ### Added
+- [`no-default-export`] rule ([#889], thanks [@isiahmeadows])
```

Comment:
Don’t forget to add `[@isiahmeadows]: https://github.com/isiahmeadows` to the bottom of this file.

Reply:
Oh yeah...oops